sphinx_automodapi/automodsumm.py

```diff
@@ -85,6 +85,7 @@ class members that are inherited from a base class. This value can be
 
 import abc
 import inspect
+import logging
```

Comment:
We already have a `from sphinx.util import logging` a few lines below, which I think is the recommended way.
Could you use it instead and put the `logging.getLogger` call in each function when it is needed ? (As already done in some cases, not sure why some others were missing).
```suggestion
```

Reply:
Done!